clr-k8s-examples/containerd-setup/configure-containerd.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+VERISION="1.2.2"
+echo "Install Containerd ${VERSION}"
+wget -q https://storage.googleapis.com/cri-containerd-release/cri-containerd-${VERSION}.linux-amd64.tar.gz
```

Comment:
Let's not do this but update the version in Clear. I'm not aware of blockers for that. Do you know any?

Reply:
Can I *just* install containerd?

Reply:
Clear already has two versions of it available. Neither are good for runtimeclass but my feedback was that we should do the update there so that pulling the third version is not needed.

Reply:
@egernst I agree with @mythi . Let us just ensure that the containerd we ship with clearlinux works with runtime class. That will ensure that it is updated as part of swupd. Otherwise it has to be maintained and updated independently.

/cc @bryteise